x-pack/plugins/ml/server/models/data_recognizer/modules/network/ml/high_count_by_destination_country.json

```diff
@@ -0,0 +1,35 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "A machine learning job detected an unusually large spike in network activity to one destination country in the network logs. This could be due to unusually large amounts of reconnaissance or enumeration traffic. Data exfiltration activity may also produce such a surge in traffic to a destination country which does not normally appear in network traffic or business work-flows. Malware instances and persistence mechanisms may communicate with command-and-control (C2) infrastructure in their country of origin, which may be an unusual destination country for the source network.",
```

Comment:
To match the style of the descriptions in the other security modules, I'd suggest rewording the first part of the description here to something like:

```
Security: Network - Looks for unusually large spikes in network activity to one destination country in the network logs.
```